### PR TITLE
[8.0][IMP] stock_move_backdating module

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

File: stock_move_backdating/README.rst
Comment (elicoidal) [inline stock_move_backdating/README.rst:37]: below please modify to https (x3)

File: stock_move_backdating/__openerp__.py
Comment (elicoidal) [inline stock_move_backdating/__openerp__.py]: I think it is better to keep it 2015 or 2015-2016 (in case of copyright claims you want to state that you were already copyright holder in 2015 right?)
Remove the (c) signs (nice to have)

File: stock_move_backdating/models/stock_move.py
Comment (elicoidal) [inline stock_move_backdating/models/stock_move.py:25]: I always have a difficulty here with this kind of modules. Not sure this is valid anymore in v8 but what if there is a Physical inventory/stock taking already approved for the product.
What would be the impact of the change in the date in the result of the stock taking?

File: stock_move_backdating/README.rst
Comment (tafaRU) [reply stock_move_backdating/README.rst:37]: Done

File: stock_move_backdating/__openerp__.py
Comment (tafaRU) [reply stock_move_backdating/__openerp__.py]: Done.

File: stock_move_backdating/README.rst
Comment (petrus-v) [inline stock_move_backdating/README.rst:36]: I've the feeling this images paragrapher is useless for this module, @Yajo can you give us an highlight of the purpose of this here?

Comment (elicoidal) [reply stock_move_backdating/README.rst:36]: if @petrus-v you use the OCA logo so you need to give the credit so keep it

Comment (petrus-v) [reply stock_move_backdating/README.rst:36]: @elicoidal ok thanks!

**Comment by petrus-v** [comment]
It may have a conflict with another module as the date is not shown on the wizard on runbot (maybe directly depends on stock module may fix this issue?)

Otherwise I've test it locally LGTM 👍

**Comment by mFlayyan** [comment]
tested on runbut and  just wondering , where is the field of "date_backdating" in the view posted bellow? :
screenshot from 2016-10-04 17-27-07

File: stock_move_backdating/README.rst
Comment (yajo) [reply stock_move_backdating/README.rst:36]: Well, not really.

This paragraph's purpose is correctly to give attribution to icon/image creator, mostly for cases where the license requires so.

However, OCA will be a direct owner of this addon once it gets merged, so giving credits to yourself is a bit recursive. If you only use the OCA icon, then this section can be considered a placeholder that you can simply remove.

**Comment by eLBati** [comment]
@mFlayyan I don't know what's wrong with runbot.
I tested locally and it's ok for me

**Review by eLBati — APPROVED** [review]

**Review by archetipo — APPROVED** [review]

**Comment by tafaRU** [comment]
Someone of @OCA/logistics-maintainers team could please merge this?

Thank you in advance.

**Review by lasley — APPROVED** [review]
LGTM code, no test

**Pull request merged**
